tests/dummy/app/controllers/index.js

```diff
@@ -1,30 +1,46 @@
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { A } from '@ember/array';
```

Comment:
These changes are handled in #87 so they will be merged to master soon.

Reply:
Ok, I pushed that by mistake...